core/services/pipeline/task.bridge.go

```diff
@@ -54,32 +68,49 @@ func (t *BridgeTask) Run(ctx context.Context, meta JSONSerializable, inputs []Re
 		)
 	}
 
-	requestData := withMeta(t.RequestData, metaMap)
-	if t.IncludeInputAtKey != "" && len(inputs) > 0 {
-		requestData[t.IncludeInputAtKey] = inputs[0].Value
+	requestData = withMeta(requestData, metaMap)
+	if t.IncludeInputAtKey != "" {
+		logger.Warnw(`The "includeInputAtKey" parameter on Bridge tasks is deprecated. Please migrate to variable interpolation syntax as soon as possible (see CHANGELOG).`,
```

Comment:
Is there a followup task to ✂️  this?

Reply:
It'll be a slightly longer-term thing to push this deprecation through, but luckily only one customer is using it right now. Adding a ticket